### PR TITLE
Update the stable CoreDNS version to v1.7.0

### PR DESCRIPTION
**Pull request by rajansandeep**
Updates from v1.6.9 to v1.7.0
Also updates the corresponding migration library to support v1.7.0

### CODE REVIEW

File: coredns/channels/packages/coredns/1.7.0/Corefile [hunk 1]
```diff
@@ -10,7 +10,9 @@
        ttl 30
```
Comment (justinsb) [inline coredns/channels/packages/coredns/1.7.0/Corefile:10]: Meta: assuming 1.6.9 works, can we keep it available for users that choose it ?

**Comment by justinsb** [comment]
LGTM, but can we / should we keep 1.6.9 available?

**Comment by rajansandeep** [comment]
Currently, I've been adding the versions that are being released with upstream k8s (CoreDNS 1.7.0 will be shipped with 1.19 etc.), but I can certainly keep 1.6.9.

**Comment by justinsb** [comment]
> Currently, I've been adding the versions that are being released with upstream k8s (CoreDNS 1.7.0 will be shipped with 1.19 etc.),

Ah - that's a reasonable strategy.  If no version of k8s shipped with 1.6.9, then I don't think we need to keep it at this stage.  nce we get everyone using the operator, then we should expect that some people will pin versions, and once we support a version we should not generally stop supporting it.

It's easy enough to add 1.6.9 back if we do need it, so approving :-)

/approve
/lgtm

Thanks @rajansandeep :-)

**Pull request merged**